File: atcoder.jp/contests/abc183/tasks/abc183_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const relu = (x: number) => (x >= 0 ? x : 0);

const x = +input();

console.log(relu(x));

File: atcoder.jp/contests/abc183/tasks/abc183_a/main.rs
use proconio::input;

fn main() {
    input! {
        x: i32,
    }

    let ans = relu(x);

    println!("{}", ans);
}

fn relu(x: i32) -> i32 {
    if x >= 0 {
        return x;
    }
    0
}
